java/templates/libraries/okhttp-gson/ApiClient.mustache

```diff
@@ -22,11 +22,6 @@ import org.joda.time.DateTime;
 import org.joda.time.LocalDate;
 import org.joda.time.format.DateTimeFormatter;
 {{/joda}}
-{{#threetenbp}}
```

Comment:
Do we need this change? Doesn't it automatically do it because of the conditional? Or is this template file old for some reason?

Reply:
Yeah, I think in theory it's not strictly necessary. I didn't see much reason to keep it around though either.